backend/src/test/java/ca/bc/gov/nrs/vdyp/backend/endpoints/v1/Hcsv_Vdyp7_Comparison_Test.java

```diff
@@ -168,7 +169,7 @@ void testVdyp8VersusVdyp7YieldTables() throws IOException {
 	static final Pattern IGNORE_COLUMNS = Pattern
 			.compile("PRJ_TPH|PRJ_LOREY_HT|PRJ_DIAMETER|PRJ_BA|PRJ_(SP\\d_)?VOL_(?:D|DW|DWB|CU|WS)");
 	static final Pattern IGNORE_COLUMNS_EXCEPT_LH = Pattern
-			.compile("PRJ_TPH|PRJ_DIAMETER|PRJ_BA|PRJ_(SP\\d_)?VOL_(?:D|DW|DWB|CU|WS)");
+			.compile("PRJ_TPH|PRJ_DIAMETER|PRJ_BA|PRJ_SCND_HT|PRJ_(SP\\d_)?VOL_(?:D|DW|DWB|CU|WS)");
```

Comment:
Should really add to the comment above these explaining that PRJ_SCND_HT is here for a different reason not related to VDYP-604.  

Maybe we should have a ticket  covering the problem of intentional VDYP7/VDYP8 differences and reference it here?